### DISCUSSION TITLE
Add a autostart chia gui with default walet when PC restart

### DISCUSSION THREAD

**Discussion by sykmer** [Chia-Network/chia-blockchain#8053] [Ideas]
Hello,

Can you add a autostart chia gui with default walet when PC restart ?
This can be very helpfull when PC (windows) restarts.

Thank you in advance and best regards.

**Comment by rappdigit** [comment]
On Linux I do it with a script as a service using the screen utility.

`
sudo -u username screen -AmdS chia_admin;


screen -S chia_admin -p 0 -X stuff "cd $dir_chia^M . ./activate^M chia start harvester^M" 
`
